language-service/src/parser/jsonParser.ts

```diff
@@ -818,6 +829,31 @@ export class ObjectASTNode extends ASTNode {
 			});
 		}
 	}
+
+	protected validateBestMatch(schema: JSONSchema, validationResult: ValidationResult): void {
+		if (schema.firstProperty && schema.firstProperty.length) {
+			if (this.properties && this.properties.length) {
+				let firstProperty = this.properties[0];
+				let firstPropKey: string = firstProperty.key.value;
+				let contained: boolean = false;
+				schema.firstProperty.forEach(listProperty => {
+					if (listProperty == firstPropKey) {
+						contained = true;
```

Comment:
break?

Reply:
break doesn't work with `forEach`.  I'll change the loop structure to allow early termination